app/models/call_for_paper.py

```diff
@@ -41,11 +40,10 @@ def serialize(self):
         return {
             'id': self.id,
             'announcement': self.announcement,
-            'start_date': self.start_date.strftime('%m/%d/%Y') if self.start_date else '',
-            'starts_at': self.start_date.strftime('%H:%M') if self.start_date else '',
-            'end_date': self.end_date.strftime('%m/%d/%Y') if self.end_date else '',
-            'ends_at': self.end_date.strftime('%H:%M') if self.end_date else '',
-            'timezone': self.timezone,
+            'starts_at': self.starts_at.strftime('%m/%d/%Y') if self.starts_at else '',
```

Comment:
How can there be two `starts_at` keys in a dict? It will just be overwritten by the second `starts_at`. Same for `ends_at`.